skel-usb.c

```diff
@@ -357,7 +367,7 @@ static int load_code_from_flash(AsebaVMState *vm) {
 
 int load_settings_from_flash(void) {
 	// Max size 95 int, min 1 int
-	COMPILATION_ASSERT(sizeof(settings) < ((INSTRUCTIONS_PER_ROW*3) - 2));
+	COMPILATION_ASSERT(sizeof(settings) < ((INSTRUCTIONS_PER_ROW*6) - 2));
```

Comment:
did you carefully check what you are doing here ? IIRC this check was put to specifically disallow that